files-community/Files
# Fix: Fixed an issue where the Properties window had the wrong icon

## Summary

As the title.

## Task Checklist

_N/A_

## Steps To Validate Changes

1. Launch the app
2. Open a properties window
3. See window icon from the taskbar

## Known Issues

_N/A_

## PR Checklist

- [ ] **Development**: _N/A_

## Screenshots

Before:
![image](https://github.com/files-community/Files/assets/62196528/27dd4041-bceb-4f9f-bc18-634b614d43c8)

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.
